teuthology/task/ansible.py

```diff
@@ -0,0 +1,225 @@
+import logging
+import requests
+import os
+import pexpect
+import yaml
+
+from cStringIO import StringIO
+from tempfile import NamedTemporaryFile
+
+from teuthology.exceptions import CommandFailedError
+from teuthology.repo_utils import fetch_repo
+
+from . import Task
+
+log = logging.getLogger(__name__)
+
+
+class LoggerFile(object):
```

Comment:
That doesn't explain the why though.  Why can't ansible use the standard logger?